src/pages/claims/claim-list.tsx

```diff
@@ -1,6 +1,6 @@
-import { NavLink } from 'react-router-dom';
+import { Link } from 'react-router-dom';
```

Comment:
Why change to `Link`? Shouldn't it be changed in other places as well?

Reply:
I accidentally used NavLink previously, NavLink is for nav menu items (it has extra props for adding active classes etc)